### DISCUSSION TITLE
Debug wait time 5000 configurable?

### DISCUSSION THREAD

**Discussion by avucic** [mfussenegger/nvim-dap#1147] [General]
https://github.com/mfussenegger/nvim-dap/blob/9adbfdca13afbe646d09a8d7a86d5d031fb9c5a5/lua/dap/session.lua#L1248 Can this  be configurable? I have issue when I try to connect to Docker. Sometimes 5000 is not enough. Thanks

**Comment by mfussenegger** [comment]
Changing it to make it configurable. See https://github.com/mfussenegger/nvim-dap/pull/1192
